pkg/services/ngalert/models/alert_rule.go

```diff
@@ -170,6 +170,14 @@ func (alertRule *AlertRule) GetLabels(opts ...LabelOption) map[string]string {
 	return labels
 }
 
+func (alertRule *AlertRule) GetEvalCondition() Condition {
+	return Condition{
```

Comment:
Not blocking:
It's confusing that a `Condition` has a `Condition` field. What do you think about renaming one of these fields?

Reply:
agree. I would rather rename the struct as it's not just a condition, but I could not figure out a better name. The renaming field is more straightforward but a little bit more confusing as we use it as DTO and ser\deser. 

I would appreciate any ideas on naming

Reply:
OrgID apparently required by Datasource cache when you resolve a datasource.